cpanfile

```diff
@@ -3,3 +3,5 @@ requires 'File::Basename';
 requires 'Data::Dumper';
 requires 'XML::Writer';
 requires 'IO::File';
+requires 'XML::LibXML';
+requires 'XML::LibXML::XPathContext';
```

Comment:
looks like you are not actually using these in the context of the driver, only within the SUT so this shouldn't be required, right?